src/burnchains/mod.rs

```diff
@@ -102,8 +99,6 @@ pub struct BurnchainParameters {
     chain_name: String,
     network_name: String,
     network_id: u32,
-    first_block_height: u64,
```

Comment:
Please revert this.  The first block height and first header hash are very much burnchain _parameters_.  You should instead update the constructors for `BurnchainParameters` to take a first block height and first block hash.

Reply:
Some background: a `BurnchainParameters` instance is meant to be a set of immutable constants that uniquely identify one of the ways the Stacks node can attach to a burnchain.  Once we release mainnet, the first burnchain block height and first burnchain block hash on mainnet Bitcoin will become protocol constants for mainnet Stacks.

Reply:
Just updating this comment from the architecture meeting.  In general, we'd like to move away from using `first_block_hash` altogether since (1) the SPV client will have all headers available anyway, and (2) the code shouldn't depend on knowing `first_block_hash` in the first place.  But, `first_block_height` is definitely an immutable protocol parameter, and should live in `BurnchainParameters`.

Reply:
Address with latest, however we had to keep the `first_block_hash`, required for initializing the Chainstate and the SortitionDB